R/generate_simulated_data.R

```diff
@@ -291,6 +340,47 @@ generate_simulated_data <- function(r_in_weeks = # nolint
     )
   }
 
+  log_r_site <- spatial_rt_process(
+    log_r_state_week,
+    corr_function,
+    corr_fun_params,
+    phi_rt,
+    sigma_eps
+  )
+  # Auxiliary Site
+  if (aux_site_bool) {
+    log_r_site_aux <- matrix(
+      data = 0,
+      nrow = 1,
+      ncol = ncol(log_r_site)
+    )
+    delta <- matrix(
+      data = 0,
+      nrow = 1,
+      ncol = ncol(log_r_site)
+    )
+    delta[1] <- rnorm(
+      n = 1,
+      mean = 0,
+      sd = sqrt(scaling_factor) * sigma_eps
+    )
+    n_t <- seq_len(ncol(log_r_site))
```

Comment:
This is creating a vector when I think you want it to create a scalar equant to the number of weeks (in which case, I would just use the previously defined `n_weeks`